js/apps/admin-ui/src/components/dropdown-panel/DropdownPanel.tsx

```diff
@@ -60,12 +47,13 @@ const DropdownPanel: React.FC<DropdownPanelProps> = ({
   }, [setSearchDropdownOpen]);
 
   return (
-    <div ref={dropdownRef}>
+    <span ref={dropdownRef}>
```

Comment:
Why make this a span?

Reply:
yeah, thanks @jonkoops , I'm using <span> here because <div> was moving the refresh button to the next line, which was wrong.